meilenstein_03/README.md

```diff
@@ -20,7 +20,7 @@
 | --- | --------- |
 |Arbeit mit PlantUML     |Bestimmung der Datentypen gewisser Attribute und Methoden           |
 |Entwurf der Klassen aufgrund guter Vorarbeit (CRC-Karten)|In manchen Gruppen musste die bei den CRC-Karten überlegte Struktur nochmals umstrukturiert werden|
-|Gute Kommunikation innerhalb der Modulgruppen||
+|Gute Kommunikation innerhalb der Modulgruppen|Interfaces falsch verstanden rip|
```

Comment:
xd schreib das mal noch nicht rein, da gibt es finde ich verschiedene Ansichten

Reply:
Habe erstmal alles bezüglich des Interfaces in Klammern gesetzt